src/main/scala/beam/utils/scenario/generic/readers/PlanElementReader.scala

```diff
@@ -90,6 +92,7 @@ object XmlPlanElementReader extends PlanElementReader {
     planElementIdx: Int
   ): PlanElement =
     PlanElement(
+      tripId = "None",
```

Comment:
I think it is better to use empty string for no-value, or, event better, to use Option[String] as field type and use None